coralproject/cay
# Emma's designs round 1

What does this PR do?

Brings trust up to date (partly) with Emma's designs

How do I test this PR?

Go to search creator for trust, look around at the new layout, click on a name, see that we now have a separate page for user details
